packages/embed/src/setups/loadIFRAME.js

```diff
@@ -0,0 +1,30 @@
+import createElement from './createElement';
+
+export default function loadIFRAME(src) {
+  return new Promise((resolve, reject) => {
+    document.body.appendChild(
```

Comment:
As a best practice, consider using a [documentFragment](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment) when adding more than 1 element to the DOM in succession.

Reply:
Never know about it, that's a very good call.

Reply:
Fixed in `createElement.js`, because it is the place where we create multiple children.

What do you think? Should `createElement` returns a `DocumentFragment` instead of the concrete element?

Reply:
A document fragment would be useful if the parent element is already added to the DOM. If the parent is not part of the DOM or has been removed from the DOM, it's fine to leave it as-is.
e.g. 

```js
const div = document.getElementById('myDiv'); // This element is on the DOM. Adding multiple children to it should always be done using a document fragment.
const newDiv = document.createElement('div'); // This element is not on the DOM so adding multiple children can be done directly or optionally with a document fragment.
```